motorsd/include/daemon_utils.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MOTOR_COMMAND_PARSER_H
+#define MOTOR_COMMAND_PARSER_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+typedef struct {
+    uint8_t motorIndex;
+    uint16_t numOfSteps;
+    uint8_t speed;
+    uint8_t checksum;
+} MotorCommand;
+
+MotorCommand parseMotorCommand(const uint8_t* byteOrder, size_t size);
```

Comment:
if this method is public API it should be prefixed with module name, such as `motorsd_parse_motor_command`